net/third_party/quiche/src/quiche/quic/core/quic_received_packet_manager_test.cc

```diff
@@ -354,7 +354,9 @@ TEST_F(QuicReceivedPacketManagerTest, AckSentEveryNthPacket) {
   }
 }
 
-TEST_F(QuicReceivedPacketManagerTest, AckDecimationReducesAcks) {
+// Note: Disabled in Cobalt because this test assumes
```

Comment:
Is it easy to change this test to use `kMaxRetransmittablePacketsBeforeAck` directly or assume a different number (50)? And what about the other for the other tests? It seems to me that we should be trying to preserve these tests in some fashion as they actually test the change

Reply:
These tests have many constants that are one time to multiples of that constant, so changing it to use the constant means changing the tests quite extensively, making rebases much harder. It would be much better to make that change in upstream, and we're quite behind from upstream here.

Reply:
Is there another way we can test this further? Right now we only test setting it in the black box test, but don't actually check if the setting took effect.

Reply:
> What would be easier... is to make kMaxRetransmittablePacketsBeforeAck a configurable parameter in the QuicReceivedPacketManager.

Saw this after my previous comment, that sounds good

Reply:
Done.

Reply:
Automatic testing whether the connection options / client connection options take effect is not easily done, because we don't have test infrastructure with a QUIC server. We can make it, because I think there are QUICHE tests that make a thread with a QUIC server, but expanding that to include the Cobalt plumbing, and then counting somewhere in the stack is nontrivial.